poutyne/framework/experiment.py

```diff
@@ -317,7 +502,45 @@ def train(self,
             if tensorboard_writer is not None:
                 tensorboard_writer.close()
 
-    def load_best_checkpoint(self, *, verbose=False):
+    def load_checkpoint(self, checkpoint, *, verbose=False):
+        """
+        Loads the attribute model's weights with the weights at a given checkpoint epoch.
+
+        Args:
+            checkpoint (Union[int, str]): If int, will load the checkpoint of the given epoch number.
+                If 'best', will load the best weights according to ``monitor_metric`` and ``monitor_mode``.
+                If 'last', will load the last model checkpoint. If int, will load the checkpoint of the
```

Comment:
The last sentence repeats the first.

Reply:
Fixed by 8ec103d03c8374f26e8acb7e2e40f07cfc68953c